.dotnet.azure/sdk/openai/Azure.AI.OpenAI/src/Azure.AI.OpenAI.csproj

```diff
@@ -5,7 +5,7 @@
       Azure OpenAI's official extension package for using OpenAI's .NET library with the Azure OpenAI Service.
     </Description>
     <AssemblyTitle>Azure.AI.OpenAI Client Library</AssemblyTitle>
-    <VersionPrefix>2.0.0</VersionPrefix>
```

Comment:
We increment the minor version when we add new features. Therefore, the next GA version will be 2.1.0, instead of 2.0.1.

We would release 2.0.1 if we needed to release bug fixes without adding any new features.